src/Components/PaperEditor/TranscriptsContainer/Paragraphs/add-annotations-to-words-in-paragraphs.js

```diff
@@ -9,18 +9,16 @@ const isAnnotationInWord = (annotation, word) => {
 
 const wordsWithAnnotations = (words, annotations) => {
 
-  if (annotations) {
```

Comment:
This has been moved up, because you would want to determine if annotations is present at all, otherwise you wouldn't call the function at all.